include/service/jobs/sites/site_activate_job.js

```diff
@@ -0,0 +1,105 @@
+
+//dependencies
+var async = require('async');
+var util  = require('../../../util.js');
+
+module.exports = function SiteActivateJobModule(pb) {
+    var GLOBAL_PREFIX = 'global';
```

Comment:
get this value from the SiteService so the value is only hard coded in one place